wsynphot/data/base.py

```diff
@@ -16,6 +16,8 @@
 ALPHA_LYR_PATH = os.path.join(os.path.dirname(__file__), 'calibration',
```

Comment:
this should live in the data directory with the filter sets.

Reply:
Changed that